component/service/src/main/java/org/exoplatform/social/service/malwareDetection/connector/MalwareDetectionFilesConnector.java

```diff
@@ -50,24 +52,38 @@ public MalwareDetectionFilesConnector(InitParams initParams, FileService fileSer
         }
       }
     } catch (Exception e) {
-      LOGGER.error("Error when trying to get the infected item informations from fileChecksum={}", fileChecksum, e);
+      long endTime = System.currentTimeMillis();
+      LOGGER.error("service={} operation={} parameters=\"fileChecksum:{}\" status=ko duration_ms={} error_msg=Error when trying to get the infected items:{}",
```

Comment:
```suggestion
      LOGGER.error("service={} operation={} parameters=\"fileChecksum:{}\" status=ko duration_ms={} error_msg=\"Error when trying to get the infected items:{}\"",
```
https://community.exoplatform.com/portal/g/:spaces:exo_itop/exo_itop/wiki/SPEC_-_logging_for_monitoring

Reply:
updated